lib/components/dropdown-item.vue

```diff
@@ -1,31 +1,21 @@
 <template>
     <b-link class="dropdown-item"
             role="menuitem"
```

Comment:
I;'m leaning more towards a new component `b-dropdown-item-btn` now, since removing the default `href` of `#` from `b-dropdown-item` might be breaking for some users (if they are expecting `a` links (although I don't think functionality would be affected, they would just get a button instead of a link)